docker-compose.yml

```diff
@@ -97,6 +97,23 @@ services:
       - serve
       - --enable-console
 
+  frontend:
+    container_name: eosrate_frontend
+    build:
+      context: ./services/frontend
+      dockerfile: ./Dockerfile
+    image: eosrate/frontend
+    depends_on:
+      - postgres
+      - hasura
+      - hapi
+      - pgweb
```

Comment:
I don't think this service requires `pgweb`

Reply:
@danazkari thanks for the explanation, I will remove those dependencies.